pkg/semrel/github_test.go

```diff
@@ -103,10 +108,7 @@ func githubHandler(w http.ResponseWriter, r *http.Request) {
 
 func getNewGithubTestRepo(t *testing.T) (*GitHubRepository, *httptest.Server) {
 	repo, err := NewGitHubRepository(context.TODO(), "", "owner/test-repo", "token")
```

Comment:
Why don't we use `require := require.New(t)` in this test?

Reply:
No real reason. I just kind of eyeballed which tests had enough assertions to merit the creation of the require variable

Reply:
okay, then let's leave it how it is 👍